package.json

```diff
@@ -126,23 +128,25 @@
     "instantsearch.js": "3.1.0",
     "jest": "23.6.0",
     "jest-preset-angular": "6.0.2",
-    "ng-packagr": "3.0.6",
+    "ng-packagr": "4.7.0",
     "node-sass": "4.11.0",
     "prettier": "1.13.7",
     "raw-loader": "1.0.0",
     "readline-sync": "1.4.9",
     "replace-in-file": "3.4.3",
-    "rxjs": "5.5.12",
+    "rxjs": "6.4.0",
     "semver": "5.6.0",
     "shelljs": "0.8.3",
     "style-loader": "0.23.1",
     "ts-loader": "5.3.3",
     "tsickle": "0.34.0",
+    "tslib": "1.9.0",
```

Comment:
this is a new peer dependency required by ng-packagr@4.7.0
https://cdn.jsdelivr.net/npm/ng-packagr/package.json